### PR TITLE
CLOUDP-115072: Adds input to filter orgs and type to filter for projects

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Adds input to filter orgs when users have more than 500 and fix type to filter projects

_Jira ticket:_ CLOUDP-115072

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

## Further comments

Detected more than 500 orgs
Screenshot 2022-07-29 at 15 34 45

Input ID
Screenshot 2022-07-29 at 15 35 56

Input name filter
Screenshot 2022-07-29 at 15 37 25

### CODE REVIEW

File: internal/cli/default_setter_opts.go [hunk 1]
```diff
@@ -100,46 +98,39 @@ func (opts *DefaultSetterOpts) projects() (pMap map[string]string, pSlice []stri
...
 	}
 
-	return pMap, pSlice, nil
+	return ids, names, nil
 }
 
 // Orgs fetches organizations and returns then as a slice of the format `nameIDFormat`,
 // and a map such as `map[nameIDFormat]=ID`.
 // This is necessary as we can only prompt using `nameIDFormat`
 // and we want them to get the ID mapping to store on the config.
```
Comment (Razeer123) [inline internal/cli/default_setter_opts.go:118]: [Q] Is this comment still valid after code changes?

File: internal/cli/default_setter_opts.go [hunk 2]
```diff
@@ -211,45 +202,82 @@ func (opts *DefaultSetterOpts) OrgExists(id string) bool {
...
-		opts.OrgID = oMap[oSlice[0]]
+	return opts.selectOrg(orgs)
+}
+
+func (opts *DefaultSetterOpts) manualOrgID() error {
+	p := &survey.Confirm{
+		Message: "Do you want to enter the Org ID manually?",
+	}
+	manually := true
+	if err2 := telemetry.TrackAskOne(p, &manually); err2 != nil {
```
Comment (Razeer123) [inline internal/cli/default_setter_opts.go:257]: [Q] Why is it `err2`? Why not `err`?

Comment (fmenezes) [reply internal/cli/default_setter_opts.go:257]: Our linters for not allow variable hiding, `err` is already defined

Comment (fmenezes) [reply internal/cli/default_setter_opts.go:254]: copy/paste

**Review by Razeer123 — DISMISSED** [review]

**Comment by Razeer123** [comment]
LGTM

**Review by Razeer123 — DISMISSED** [review]

**Comment by fmenezes** [comment]
@dianchenghu if you have more insights I can implement them on another PR

**Comment by dianchenghu** [comment]
Add design & notes as reference: https://www.figma.com/file/RYOrTIaf2ksQsVT0Sp6RYA/Atlas-CLI-onboarding?node-id=2314%3A5872

**Review by Razeer123 — APPROVED** [review]

**Pull request merged**